test/Microsoft.AspNetCore.Mvc.Razor.Test/Internal/ViewComponentTagHelperDescriptorFactoryTest.cs

```diff
@@ -87,6 +86,13 @@ public static TheoryData TypeData
         [MemberData(nameof(TypeData))]
         public void GetCSharpTypeName_ReturnsCorrectTypeNames(Type type, string expected)
         {
+            // Arrange
+            // Work around problems serializing / deserializing this ref type.
+            if (type == null)
+            {
+                type = typeof(ViewComponentTagHelperDescriptorFactoryTest).GetMethod("MethodWithOutParam").GetParameters().First().ParameterType;
+            }
```

Comment:
@ajaybhargavb did you file a bug about the serialization / deserialization problems with the `out List<char*[]>` parameter's `Type`? Would like to reference it here.